mapbox/services/uploads.py

```diff
@@ -62,7 +65,12 @@ def stage(self, fileobj, creds=None):
             region_name="us-east-1")
 
         s3 = session.resource('s3')
-        res = s3.Object(creds['bucket'], creds['key']).put(Body=fileobj)
+        # We'll use Bucket.upload_file() for actual files.
+        # Progress reporting for the CLI is a TODO.
+        if hasattr(fileobj, 'name') and os.path.exists(fileobj.name):
+            s3.Bucket(creds['bucket']).upload_file(fileobj.name, creds['key'])
```

Comment:
This feels a bit funny to require an open file handle only to access the object by its path. If it weren't for the pending changes in https://github.com/mapbox/mapbox-cli-py/pull/68 (stdin bytes accumulated client side then wrapped in a file-like object) I'd say just change the API to use a path. But I think this is a good solution for now.